flytekit/configuration/platform.py

```diff
@@ -4,3 +4,7 @@
 
 URL = _config_common.FlyteRequiredStringConfigurationEntry('platform', 'url')
 INSECURE = _config_common.FlyteBoolConfigurationEntry('platform', 'insecure', default=False)
+AUTH = _config_common.FlyteBoolConfigurationEntry('platform', 'auth', default=False)
```

Comment:
Would we actually configure this? or would we get a redirect when trying to connect?

Reply:
@wild-endeavor do you know what the plan is? mind weighing in?

Reply:
Sorry what will this be?  Whether or not to use auth?

Reply:
it would be configured.  i think configuring is cleaner than trying without and then doing it automatically.  if admin returns an authorization error, I'd rather print a nice error message asking the user to enable auth.